docs/UserGuide.md

```diff
@@ -31,104 +31,238 @@
+Shows a list of all categories in LifeKeeper.<br>
+Format: `listcat`
+
+> All the categories in Lifekeeper will be listed
+
+
+#### Finding tasks by name, tag(s) or the range of DEADLINEs: `find`
+Finding all tasks containing the queried keyword in their name
+* Finds tasks whose names contain any of the given keywords.<br>
+* Format: `find tasks KEYWORD [MORE_KEYWORDS]`
+
+> * The search is not case sensitive. e.g `study` will match `Study`
+> * The order of the keywords matters. e.g. `Assignment Due` will not match `Due Assignment`
+> * Words containing the keywords will be matched e.g. `Exam` will match `Exams`
+> * Tasks matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `Shopping` will match `Clothes Shopping`
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find tasks Homework Assignment`<br>
+  Returns Any tasks with words containing `Homework`, `homework`, `Assignment`, or `assignment` in their names.
+
+Finding all tasks containing a certain tag    
+* Finds tasks which has tags of given keywords attached to it.<br>    
+* Format: `find tags KEYWORD [MORE_KEYWORDS]`   
+    
+> * The search is not case sensitive.     
+> * Only full words will be matched.    
+> * Only tags matching the EXACT keyword will be returned.    
+    
+Examples:   
+* `find tags CS2103`    
+  Returns Any tasks containing the tag `CS2103` or `cs2103` but not `CS2103T` or `CS2103 Project`.    
+    
+Finding all tasks which has deadlines falling within certain dates    
+* Finds tasks which has DEADLINEs falling between the specified range.<br>    
+* Format: `find date STARTING_DATE ENDING_DATE`    
+    
+> Date must be in the form `d MMM yyyy`, `d-MM-yyyy` or `d/MM/yyyy`
+    
+Examples:   
+`find date 1 Oct 2016 31 Oct 2016`   
+  Returns Any tasks which has DEADLINEs falling in between 1st October 2016 and 31st October 2016.<br>
+`find date 1-11-2016 31-12-2016`   
+  Returns Any tasks which has DEADLINEs falling in between 1st November 2016 and 31st December 2016.
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+#### Deleting a task: `delete`
+Deletes the selected task from Lifekeeper. Irreversible.<br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task with `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+  
 
 Examples: 
 * `list`<br>
+  `delete 1,3`<br>
+  Deletes the 1st and 3rd task in the Lifekeeper task list.
+* `find Dinner`<br>
   `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Selects the 2nd task in the results of the `find` command and then deletes it.
+
+#### Select: `select`
```

Comment:
select and do what? what is the motivation behind having a select?
 btw, below, your select category and select task thing is a little confusing.
try making it more explicit.